components/uuf-core/src/test/java/org/wso2/carbon/uuf/core/APITest.java

```diff
@@ -70,6 +75,8 @@ public void testSendRedirect() {
 
     @Test
     public void testCreateSession() {
```

Comment:
Required tests (create, destroy and get session without repeating the same test case) are already added to `InMemorySessionManagerTest` class. It is true that the `API.createSession()` simply calls the `SessionHandler.createSession()`. However, `API` should be tested treating the `API` class as an individual. Therefore we should keep this test